.asf.yaml

```diff
@@ -31,6 +31,10 @@ github:
       required_pull_request_reviews:
         dismiss_stale_reviews: false
         required_approving_review_count: 1
+      required_status_checks:
+        strict: true
+        contexts:
+          - "kvrocks ci actions / required (pull_request)"
```

Comment:
I think `pull_request` is unneeded. You can refer to https://github.com/apache/skywalking/blob/master/.asf.yaml.

Reply:
done